web-httpd-implementation.md

```diff
@@ -0,0 +1,10 @@
+# Talking points from protocol.berg
```

Comment:
Does this belong here?

Reply:
Uuh no it does not 😅

Reply:
ea4f9ec4fdba3b2af5665ba840db7db3ecbb1daa